### ISSUE TITLE
[Feature Request] Block Sync View for Dojo enabled Wallets

### ISSUE THREAD

**Issue by PuraVlda** [Samourai-Wallet/samourai-wallet-android#368]
For Wallets using Dojo Back end it would be nice to see a toggle to verify what Block Header is current from API tabvon Dojo within the Wallet.

Maybe by clicking the Samurai Logo at top left you can pull down and see what Block Your Dojo is synced too (API).  This would alleviate having to go into the maintenance tool to verify.  

Reason I thought this is I was receiving a TX and wallet did not show, apparently have an API issue in Dojo with blocks being behind 40+.

**Comment by Relaxo143** [comment]
I support this idea.

**Comment by samouraiwallet** [comment]
Good idea. I think the "networking" screen in the wallet is more appropriate for this information.
